changelogs/fragments/339-add-require-message-auth-for-radius.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_info, api_modify - Add missing attribute ``require-message-auth`` for the ``radius`` path which exists since RouterOS version 7.15 (https://github.com/ansible-collections/community.routeros/issues/338, https://github.com/ansible-collections/community.routeros/pull/339).
```

Comment:
```suggestion
  - api_info, api_modify - add missing attribute ``require-message-auth`` for the ``radius`` path which exists since RouterOS version 7.15 (https://github.com/ansible-collections/community.routeros/issues/338, https://github.com/ansible-collections/community.routeros/pull/339).
```